resources/mail/layouts/html.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+/* @var $content string Mail contents as view render result */
+?><!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+
+<html xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
```suggestion
?><!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<html xmlns="http://www.w3.org/1999/xhtml">
```
I apologize for my meticulousness, but some email clients are very picky about the layout and work according to the old more stringent standards.
I think it’s better not to create a potential rake, so that later someone steps on them.
Let's not put dubious line breaks just for the sake of beauty.